test/e2e/util/util.go

```diff
@@ -77,6 +80,21 @@ import (
 	"go.uber.org/zap/zaptest"
 )
 
+var (
+	// CiliumVersion is read from CILIUM_VERSION at runtime. When empty, the Cilium tests should skip.
+	CiliumVersion = os.Getenv("CILIUM_VERSION")
```

Comment:
- This needs to be public?
- Could we move the cilium related vars to the cilium.go file?